devops/scripts/deploy-decoupled-upstream.sh

```diff
@@ -101,6 +102,7 @@ for commit in "${commits[@]}"; do
 done
 
 echo "Executing decoupledpatch.sh"
+# shellcheck disable=SC1091
```

Comment:
recommend source instead of disable.

Reply:
Disabling here because the file doesn't exist in the directory it's being sourced from in the script, so the linter is telling you "that file doesn't exist" (which is correct, but irrelevant).